src/lib/storage.ts

```diff
@@ -32,3 +34,5 @@ export const remPrefix = (prefix: string) => {
 export const set = (key: string, value: any) => {
   localStorage.setItem(`jukelab:${key}`, JSON.stringify(value));
 };
+
+export const holidayMode = writable(false);
```

Comment:
This file is just supposed to be low level localStorage helper.

So "holiday" shouldn't go here. I also don't think you need svelte/store at all.

I'd put this in page.svelte as `ui.holiday` and pass that around

Reply:
oooo man, right on! I started here looking to persist it in localStorage, then had issues with browser/server diff and gave up lol.

still learning Svelte, but it seems like if I have this flag as a state in `desktop/+page.svelte` then I need events to change it from the menu? otherwise need to pass it down like `<Menu holiday={holiday} on:changeHoliday={something}>`?

or use `svelte/store` to have it defined in a page and have two components read/write it? 🤔

Reply:
getting this one in quick too, but let's chat/follow up on this soon!